src/templates/person.js

```diff
@@ -113,11 +113,17 @@ const Person = ({ data }) => {
             </dl>
             <dl>
               <dt>Skills:</dt>
-              {advancedSkills.map((skill, i) => (
-                <dd key={`a${i}`}> {skill};</dd>
+              {advancedSkills.map((skill, i, arr) => (
+                <dd key={`a${i}`}>
```

Comment:
Can you make the key name a bit more explicit ? (same for intermediate) 
I know you didn't do it in the first place but i missed it

Reply:
done 👍